base/src/main/java/org/aya/core/visitor/VarConsumer.java

```diff
@@ -113,7 +113,9 @@
         bound.append(param.ref());
         param.type().accept(this, Unit.unit());
       });
-      bound.removeAt(start, term.params().size());
+      for (int i = 0; i < term.params().size(); i++) {
+        bound.removeAt(start); // TODO: DynamicSeq::removeAt(int, int)
```

Comment:
Can you update Kala with this method?

Reply:
> Can you update Kala with this method?

There are usually two styles of APIs that need to specify a range:

1. Accept an index and a length;
2. Accept begin index and end index.

The original `removeAt(int, int)` is inherited from Scala's `remove`, which belongs to the first style. However, Kala's overall API design tends to the second style, which makes it a little out of place.

For this reason, I have temporarily removed this API. I need to spend some time exploring it again and reintroducing it in a new form in the future.